addons/main/script_macros_common.hpp

```diff
@@ -796,6 +796,19 @@ Macro: GVARMAIN()
     #define PREPMAIN(var1) ['PATHTO_SYS(PREFIX,COMPONENT_F,DOUBLES(fnc,var1))', 'TRIPLES(PREFIX,fnc,var1)'] call SLX_XEH_COMPILE_NEW
 #endif
 
+#ifdef RECOMPILE
+    #undef RECOMPILE
+    #define RECOMPILE recompile = 1
+#else
+    #undef RECOMPILE
```

Comment:
If RECOMPILE already is undefined, you cannot undefine RECOMPILE. This might or might not error out with Mikeros Tools too.